File: MODUL6/PraktikumC/PRAK601-2310817110004-ABDURRAHMANGILANGHARJUNA.c
#include <stdio.h>

void Matriks(int baris, int kolom, int elemen[]) {
    int matriks[baris][kolom];
    int indeksElemen = 0;

    for (int i = 0; i < baris; i++) {
        for (int j = 0; j < kolom; j++) {
            matriks[i][j] = elemen[indeksElemen];
            indeksElemen++;
        }
    }

    printf("Output:\n");
    for (int i = 0; i < baris; i++) {
        for (int j = 0; j < kolom; j++) {
            printf("%d ", matriks[i][j]);
        }
        printf("\n");
    }
}

int main() {
    int baris, kolom;
    scanf("%d %d", &baris, &kolom);

    int jumlahElemen = baris * kolom;
    int elemenMatriks[jumlahElemen];
    for (int i = 0; i < jumlahElemen; i++) {
        scanf("%d", &elemenMatriks[i]);
    }
    Matriks(baris, kolom, elemenMatriks);

    return 0;
}

File: MODUL6/PraktikumPy/PRAK601-2310817110004-ABDURRAHMANGILANGHARJUNA.py
def matriks(baris, kolom, elemen):
    matriks = [[0] * kolom for _ in range(baris)]
    indeks_elemen = 0

    for i in range(baris):
        for j in range(kolom):
            matriks[i][j] = elemen[indeks_elemen]
            indeks_elemen += 1

    return matriks

def output_matriks(matriks):
    for baris in matriks:
        print(*baris)

input_baris_kolom = input()
baris, kolom = map(int, input_baris_kolom.split())

input_elemen = input()
elemen_matriks = list(map(int, input_elemen.split()))

matriks_hasil = matriks(baris, kolom, elemen_matriks)

print("Output:")
output_matriks(matriks_hasil)
